pkg/scheduler/extender_test.go

```diff
@@ -388,3 +390,146 @@ func TestIsInterested(t *testing.T) {
 		})
 	}
 }
+
+func Test_convertToNodeNameToMetaVictims(t *testing.T) {
+	type args struct {
+		nodeNameToVictims map[string]*extenderv1.Victims
+	}
```

Comment:
you don't need this type, just use `nodeNameToVictims map[string]*extenderv1.Victims` instead of `args args` below.